package/src/core/define-options.ts

```diff
@@ -14,16 +15,16 @@ import type { Options } from "../core/types.js";
  * type Options = {
  * 		name?: string | undefined
  * };
- * 
+ *
  * export default defineIntegration({
  * 		// ...
  * 		options: defineOptions<Options>({ name: "abc" }),
  * })
  * ```
  */
-export const defineOptions = <TOptions extends Record<string, unknown>>(
-	defaults: Required<TOptions>,
-): Options<TOptions> => ({
+export const defineOptions = <TOptions extends Record<string, unknown>>({
+	schema,
+}: { schema: import("astro/zod").AnyZodObject }): Options<TOptions> => ({
```

Comment:
So... the schema is not tied at all to the type.

Sounds like a footgun. Loaded.

Reply:
xd yeah probably 😅 if you have any idea I'm open:

Reply:
I'll give it a try

Reply:
See #58